networks/calibration/explorers.md

```diff
@@ -12,7 +12,7 @@ description: >-
 
 ## FILFOX
 
-[https://calibration.filfox.info/en](https://calibration.filfox.info/en)
+[https://calibration.filfox.info/en](http://47.109.105.51/en)
```

Comment:
this situation is temporary - I'm trying to figure out what's going on with them but we should probably list both for now.

Reply:
It doesn't pass the link checker test if the link is broken and thus the PR can't be merged.. We can revert is once they fix it?